layers/xt-prod-devel-rpi5-domd/recipes-kernel/linux/linux-raspberrypi_6.12.bbappend

```diff
@@ -0,0 +1,22 @@
+require include/dt_config.inc
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
```

Comment:
WARNING: 'prefered' may be misspelled - perhaps 'preferred'?
#7: 
Add the recipe for the 6.12 kernel version and change the prefered linux
                                                          ^^^^^^^^